packages/framework/tests/Unit/Model/Order/Processing/OrderProcessorMiddleware/PersonalPickupPointMiddlewareTest.php

```diff
@@ -0,0 +1,126 @@
+use Shopsys\FrameworkBundle\Model\Store\Store;
+use Shopsys\FrameworkBundle\Model\Store\StoreData;
+use Shopsys\FrameworkBundle\Model\Store\StoreFacade;
+use Shopsys\FrameworkBundle\Model\Transport\Transport;
+use Tests\FrameworkBundle\Test\MiddlewareTestCase;
+
+class PersonalPickupPointMiddlewareTest extends MiddlewareTestCase
+{
+    public function testNoChangeWithoutPersonalPickupPointId(): void
+    {
+        $orderProcessingData = $this->createOrderProcessingData();
+
+        $personalPickupPointMiddleware = $this->createPersonalPickupPointMiddleware();
+
+        $result = $personalPickupPointMiddleware->handle($orderProcessingData, $this->createOrderProcessingStack());
+        $actualOrderData = $result->orderData;
+
+        $this->assertNull($actualOrderData->personalPickupStore);
+        $this->assertNull($actualOrderData->pickupPlaceIdentifier);
+    }
+
+    public function testUpdatedDeliveryDataByStore(): void
+    {
+        $expectedPersonalPickupPoint = 'personalPickupPoint';
+        $orderProcessingData = $this->createOrderProcessingData();
+
+        $transport = $this->createMock(Transport::class);
+        $transport->method('isPersonalPickup')->willReturn(true);
+
+        $orderProcessingData->orderInput->setTransport($transport);
+        $orderProcessingData->orderInput->addAdditionalData(PersonalPickupPointMiddleware::ADDITIONAL_DATA_PICKUP_PLACE_IDENTIFIER, $expectedPersonalPickupPoint);
+
+        $transportData = new OrderItemData();
+        $transportData->type = OrderItemTypeEnum::TYPE_TRANSPORT;
+        $orderProcessingData->orderData->items[] = $transportData;
+
+        $city = 'city';
+        $street = 'street';
+        $postcode = '12345';
+        $country = $this->createMock(Country::class);
+
+        $storeData = new StoreData();
+        $storeData->city = $city;
+        $storeData->street = $street;
+        $storeData->postcode = $postcode;
+        $storeData->country = $country;
+        $store = new Store($storeData);
+
+        $personalPickupPointMiddleware = $this->createPersonalPickupPointMiddleware($store);
+
+        $result = $personalPickupPointMiddleware->handle($orderProcessingData, $this->createOrderProcessingStack());
+        $actualOrderData = $result->orderData;
+
+        $this->assertSame($store, $actualOrderData->personalPickupStore);
+        $this->assertSame($expectedPersonalPickupPoint, $actualOrderData->pickupPlaceIdentifier);
+
+        $this->assertFalse($actualOrderData->deliveryAddressSameAsBillingAddress);
+
+        $this->assertSame($street, $actualOrderData->deliveryStreet);
+        $this->assertSame($city, $actualOrderData->deliveryCity);
+        $this->assertSame($postcode, $actualOrderData->deliveryPostcode);
+        $this->assertSame($country, $actualOrderData->deliveryCountry);
+    }
+
+    public function testPersonalPickupPointIsAdded(): void
```

Comment:
I am confused about the test name or content :thinking: 
It is called "personal pickup point is added" but the added transport's `isPersonalPickup` method should return false? And in the result, there is no store but there is just the identifier? Is it because of Packetery? Maybe I will need to discuss it further :sweat_smile:

Reply:
renamed to `testPacketeryIdentifierIsAdded`, is it clearer: